### PR TITLE
nspr 4.10.9

### PR DESCRIPTION
**Pull request by l-hedgehog**

### CODE REVIEW

File: Library/Formula/nspr.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class Nspr < Formula
   desc "Platform-neutral API for system-level and libc-like functions"
   homepage "https://developer.mozilla.org/docs/Mozilla/Projects/NSPR"
-  url "https://ftp.mozilla.org/pub/mozilla.org/nspr/releases/v4.10.8/src/nspr-4.10.8.tar.gz"
-  sha256 "507ea57c525c0c524dae4857a642b4ef5c9d795518754c7f83422d22fe544a15"
+  url "https://ftp.mozilla.org/pub/mozilla.org/nspr/releases/v4.10.9/src/nspr-4.10.9.tar.gz"
+  sha256 "4112ff6ad91d32696ca0c6c3d4abef6367b5dc0127fa172fcb3c3ab81bb2d881"
   revision 1
```
Comment (DomT4) [inline Library/Formula/nspr.rb]: Kill this line on version bumps.

**Comment by crazyhack** [comment]
❤️

File: Library/Formula/nspr.rb (hunk 1 above)
Comment (l-hedgehog) [inline Library/Formula/nspr.rb]: Oops, removed, thanks!

**Comment by mikemcquaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

**Comment by brandonfilley** [comment]
Thanks

**Pull request closed without merging**